src/3rd_party_add.c

```diff
@@ -102,7 +102,7 @@ enum swupd_code third_party_add_main(int argc, char **argv)
 	ret = third_party_add_repo(name, url);
 	if (ret) {
 		if (ret != -EEXIST) {
-			error("Failed to add repo: %s to config\n", name);
+			error("Failed to add repository: %s to config\n", name);
```

Comment:
+1 for using repository instead of repo. Let's try to use that in docs and messages.